src/screens/FeedScreen/hooks/useFeed.js

```diff
@@ -96,13 +97,13 @@ const useFeed = () => {
 
   const handleTextCountStyle = () => {
     if (totalVote > 0) {
-      return '#00ADB5';
+      return COLORS.holyTosca;
     }
     if (totalVote < 0) {
-      return '#FF2E63';
+      return COLORS.red;
     }
 
-    return '#C4C4C4';
+    return COLORS.gray1;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hard-coded color values have been replaced with references to color constants from a theme file. This is a good practice as it improves consistency across the application and makes future changes easier.
```diff
-      return '#00ADB5';
+      return COLORS.holyTosca;
-      return '#FF2E63';
+      return COLORS.red;
-    return '#C4C4C4';
+    return COLORS.gray1;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->